packages/react-components/lib/omni-panel.tsx

```diff
@@ -24,8 +24,7 @@ const useStyles = makeStyles((theme) => ({
   },
   navigationButton: {
     borderRadius: 'inherit',
-    backgroundColor: theme.mainBackground,
-    color: theme.fontColors,
+    borderColor: theme.palette.divider,
```

Comment:
I think the default border color is not the same as the divider color, so doing this would mess up the default material theming.